app/controllers/concerns/json_web_token.rb

```diff
@@ -0,0 +1,15 @@
+require 'jwt'
+module JsonWebToken
+  extend ActiveSupport::Concern
+  SECRET_KEY = Rails.application.secret_key_base
+
+  def jwt_encode(payload, exp = 7.days.from_now)
+    payload[:exp] = exp.to_i
+    JWT.encode(payload, SECRET_KEY)
+  end
+
+  def jwt_decode(token)
+    decode = JWT.decode(token, SECRET_KEY)[0]
+    HashWithIndifferentAccess.new decode
+  end
```

Comment:
Nice job so far😉. Currently, an error from the `jwt_decode` method keeps preventing your API endpoints from working properly. I think it is because there's an issue decoding your JWT token😉 hence it's I can't receive the data in JSON format😉. Kindly do well to fix the issue and ensure all your endpoints work properly before requesting a review😉. 
<img width="1089" alt="image" src="https://github.com/patriciachrysy/Kdrama-Lovers-Blog/assets/64643995/e33069ad-59c0-44df-bc73-8b325c2dc01e">

<img width="1089" alt="image" src="https://github.com/patriciachrysy/Kdrama-Lovers-Blog/assets/64643995/82be5134-c572-42bb-8d98-f0253563ef6a">

Reply:
I'm getting the following feedback from my partner:

- I tested yesterday and I was working just fine
- The problem he has is that he is not passing the jwt token when he tests
- Ask him if he authentified to get the token and then if he passed it as bearer token in the header of the request

Reply:
Please confirm the above before I make another review

Reply:
Hello, @jkanyi-web 👋🏼.
I actually do not think that's the issue. but nevertheless, 
I'd suggest you kindly add a step by step instruction or guide to how to work with your API. this will make it easier for the next reviewer to work with😉.

But before making any submission, Kindly confirm by yourself that everything works fine.

Reply:
Hey @Ibizugbe, I've discovered where the issue was after manual testing and fixed it. Thanks